gateway-ha/src/test/java/io/trino/gateway/ha/clustermonitor/TestClusterStatsMonitor.java

```diff
@@ -56,6 +56,12 @@ public void testJdbcMonitor()
         testClusterStatsMonitor(ClusterStatsJdbcMonitor::new);
     }
 
+    @Test
+    public void testInfoApiMonitor()
+    {
+        testClusterStatsMonitor(ignored -> new ClusterStatsInfoApiMonitor());
```

Comment:
Looks like this test doesn't ensure the expected status for `/v1/status`. It passes even if ServerInfo's field name differ, e.g. starting -> test

Reply:
Good catch. I actually intended to use the `/v1/info` endpoint. I retested manually after updating the `ServerInfo` constructor but before changing `ClusterStatsInfoApiMonitor.monitor` to use `/v1/info`, and the test now fails as expected when the response does not contain the `starting` field.